### PR TITLE
UI rebrush

### PR DESCRIPTION
**Pull request by CivanErbay**

### CODE REVIEW

File: frontend-vue/public/index.html [hunk 1]
```diff
@@ -1,10 +1,13 @@
...
-    <meta http-equiv="X-UA-Compatible" content="IE=edge" />
-    <meta name="viewport" content="width=device-width,initial-scale=1.0" />
-    <link rel="icon" href="<%= BASE_URL %>favicon2.svg" />
+    <meta charset="utf-8"/>
+    <meta http-equiv="X-UA-Compatible" content="IE=edge"/>
+    <meta name="viewport" content="width=device-width,initial-scale=1.0"/>
+    <link rel="icon" href="<%= BASE_URL %>favicon2.svg"/>
+    <link rel="preconnect" href="https://fonts.googleapis.com">
+    <link rel="preconnect" href="https://fonts.gstatic.com" crossorigin>
+    <link href="https://fonts.googleapis.com/css2?family=Patrick+Hand&display=swap" rel="stylesheet">
```
Comment (manuliner) [inline frontend-vue/public/index.html:10]: please use local hosted google fonts

File: frontend-vue/src/main.js [hunk 2]
```diff
@@ -1,48 +1,48 @@
...
 import router from "./plugins/router";
-import vuetify from "./plugins/vuetify";
-import store from "./stores";
-
-// eslint-disable-next-line no-unused-vars
-import base from "./plugins/base";
-// eslint-disable-next-line no-unused-vars
-import custom from "./plugins/custom";
-
+import VueCookies from "vue3-cookies";
```
Comment (manuliner) [inline frontend-vue/src/main.js:4]: do we need cookies for technical reasons? we could use local storage for persistent data.

**Review by manuliner — CHANGES_REQUESTED** [review]

File: frontend-vue/public/index.html (hunk 1 above)
Comment (CivanErbay) [reply frontend-vue/public/index.html]: font now locally embedded

File: frontend-vue/src/main.js (hunk 2 above)
Comment (CivanErbay) [reply frontend-vue/src/main.js]: cookie-lib deleted // local storage should be enough

**Review by manuliner — APPROVED** [review]

**Pull request merged**